src/sources/file.rs

```diff
@@ -204,7 +211,9 @@ pub struct FileConfig {
     ///
     /// If not specified, files will not be removed.
     #[serde(alias = "remove_after", default)]
-    #[configurable(metadata(docs::examples = "example_seconds()"))]
+    #[configurable(metadata(docs::examples = 0))]
+    #[configurable(metadata(docs::examples = 5))]
+    #[configurable(metadata(docs::examples = 60))]
     pub remove_after_secs: Option<u64>,
```

Comment:
I believe unit type is still something we can set, which I've been doing for the ones that are Option<>

Reply:
Setting it here didn't render the ui any differently, but I can include it for completeness if you'd prefer.

Reply:
Shouldn't  this be a duration?

Reply:
See - https://github.com/vectordotdev/vector/pull/16099#discussion_r1085537874, https://github.com/vectordotdev/vector/pull/16108#discussion_r1086704228, https://github.com/vectordotdev/vector/issues/15674#issuecomment-1373731434

Reply:
Ah yeah, I remember now. That's annoying.

Reply:
ok, if it doesn't render, than I guess there is no benefit in adding it. I'll keep that in mind going forward.

Reply:
Speaking offline - it seems like this is related to there being no default value for the site to render, which is where the unit gets printed as well.

We should keep typing appropriately for the OP ui (if they're using it) and for future use if/when we improve the site 👍

Reply:
Addressed in [0e7c38b](https://github.com/vectordotdev/vector/pull/16099/commits/0e7c38bf4f326cfe4575ba418a6da0996b4a8658)